src/Plugin.php

```diff
@@ -234,6 +239,35 @@ public static function xmlrpc_methods($methods) {
     return $methods;
   }
 
+  /**
+   * @implements wp_head
+   */
+  public static function wp_head() {
+    $data = [
+      'ajaxurl' => admin_url('admin-ajax.php'),
+      'consent_version' => CORE_STANDARDS_COOKIE_CONSENT_VERSION,
+    ];
+    echo '<script>var core_standards = ' . json_encode($data) . ';</script>';
```

Comment:
1. WP has a built-in support for JavaScript variables via wp_enqueue_scripts() or a closely related function. This should be used instead.

1. Please make sure that this JS variable name is not taken yet. If I'm not mistaken, core-standards also uses gettext string translations for JavaScript, which are potentially output with the same name.

1. Options for json_encode(), as mentioned earlier in this PR.

1. Major: The constant `CORE_STANDARDS_COOKIE_CONSENT_VERSION` is not defined anywhere.

1. All of the added code in this file, including the constant definition, should be moved to a new CookieConsent class. As the code is executed on all pages, you can simply move the code as-is and create a `CookieConsent::init()` method, which you call from `Plugin::init()`.

Reply:
1. Not sure what you mean by this. Do you mean https://developer.wordpress.org/reference/functions/wp_localize_script/? If so, this is not possible, as `wp_localize_script` is working for enqueued/registered assets only. As we don't have a dedicated `.js` file this function cannot be used. This was the only way I found by the current approach. We could extract the JavaScript code still.

Reply:
I changed the JS to be enqueued instead of rendered inline so `wp_localize_script` works now 👍